tests/chainerx_tests/unit_tests/routines_tests/test_connection.py

```diff
@@ -174,6 +174,12 @@ class TestConvTensorCore(_ConvTestBase, op_utils.ChainerOpTest):
     out_dtype = 'float16'
     cover_all = False
 
+    def __init__(self):
+        # TODO(imanishi): Skip only if compute_capability < 70.
+        # Huge floating-point error often arises, if a CuDNN function does not
+        # choose a Tensor Core-based algorithm was not choosed.
```

Comment:
Could you please rewrite the comment? the meaning of the if a CuDNN ... sentence is a bit unclear.